### ISSUE TITLE
Add custom logging for character selection 

### ISSUE THREAD

**Issue by whoismattclark** [whoismattclark/lorem-tools#9]
https://developers.google.com/analytics/devguides/collection/analyticsjs/events

**Comment by whoismattclark** [comment]
@jake-nathan this isn’t terribly useful information but I was going to add it to get more experience with GA events. What’s your JavaScript level? Does this seem hard or easy for you?

**Comment by jake-nathan** [comment]
My JS level is quite limited, however, I'm down to try to tackle this.

So you want to know how many characters people are selecting each time?

**Comment by whoismattclark** [comment]
Yeah, maybe we could do a screen share and we can pair on it. Let’s do this
post launch after we’ve closed out all the other tickets.
